include/tvm/relay/qnn/attrs.h

```diff
@@ -44,14 +44,15 @@ struct RequantizeAttrs : public tvm::AttrsNode<RequantizeAttrs> {
       .set_default(-1);
     TVM_ATTR_FIELD(rounding).set_default("UPWARD")
         .describe("Defines the rounding direction when the value is midway between"
-                  "two representable values. There are two supported modes - UPWARD"
-                  "or TONEAREST. Both modes behave exactly same except at the"
+                  "two representable values. There are two 3 modes - UPWARD, TONEAREST"
+                  "or TFLITE. UP/TONEAREST modes behave exactly same except at the"
                   "midpoints between the two representable values. At the midpoint,"
                   "UPWARD rounds towards positive infinity (for example -1.5 will be"
                   "rounded to -1). TONEAREST is the standard rounding where the"
                   "value is rounded away from zero at midpoints (for example, -1.5"
                   "rounds to -2). More context can be found at following gblic manual"
```

Comment:
While you are here fix the typo gblic .. it's glibc.

Reply:
updated accordingly